examples/tv-casting-app/android/App/app/src/main/jni/com/matter/casting/support/CommissionerDeclaration.java

```diff
@@ -0,0 +1,172 @@
+/**
+ * Copyright (c) 2024 Project CHIP Authors All rights reserved.
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.matter.casting.support;
+
+import android.util.Log;
+
+/**
+ * Represents the Commissioner Declaration message sent by a User Directed Commissioning server
+ * (CastingPlayer/Commissioner) to a UDC client (Casting Client/Commissionee).
+ */
+public class CommissionerDeclaration {
```

Comment:
feels wasteful to not be able to use the similar type defined in UserDirectedCommissioning.h

Reply:
see comment below